DarkBOT.iml

```diff
@@ -11,12 +11,13 @@
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
     <orderEntry type="library" name="Maven: com.google.code.gson:gson:2.8.5" level="project" />
-    <orderEntry type="library" name="Maven: com.bulenkov:darcula:1.0.0" level="project" />
```

Comment:
Let maven install this dep. Use `mvn clean install` and it will automatically load this